scripts/utility.py

```diff
@@ -1295,7 +1295,7 @@ def generate_sprite(cat, life_state=None, scars_hidden=False, acc_hidden=False,
         # draw eyes & scars1
         eyes = sprites.sprites['eyes' + cat.pelt.eye_colour + cat_sprite].copy()
         if cat.pelt.eye_colour2 != None:
-            eyes.blit(sprites.sprites['eyes2' + cat.pelt.eye_colour2 + cat_sprite], (0, 0))
+            eyes.blit(sprites.sprites["eyes2" + cat.pelt.eye_colour2 + cat_sprite], (0, 0))
```

Comment:
What is the purpose of this change? Is this to fix an unrelated bug?  Same with the new "eye2color.png" file.

Reply:
Just a mistake when I was working on a different branch and hadn't properly changed my branches over. It should be fixed.